lib/device.js

```diff
@@ -7,7 +7,12 @@
 const util = require('util'),
     async = require('async'),
     npmlog = require('npmlog'),
+    fs = require('fs'),
+    path = require('path'),
+    mkdirp = require('mkdirp'),
+    chalk = require('chalk'),
     luna = require('./base/luna'),
+    pullLib = require('./pull'),
```

Comment:
I think it would be better if this is the last one.

Reply:
Fixed. Done.